rin_serin/week04/mission02/src/components/Card.jsx

```diff
@@ -0,0 +1,62 @@
+import styled from "styled-components";
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { Navigate } from "react-router-dom";
+import { Link, replace } from "react-router-dom";
+
+const CardDiv = styled.button`
+    margin: 10px;
+    width: 160px;
+    padding: 10px;
+    height: fit-content;
+    background: none;
+    border: none;
+    display: flex;
+    flex-direction: column;
+    box-sizing: border-box;
+`
+const MovieImg = styled.img`
+    width: 160px;
+    height: 240px;
+    border-radius: 12px;
+    transition: opacity 0.3s ease-in-out;
+    filter: ${props =>(props.hoverOn ? 'brightness(50%)' : 'brightness(100%)')};
+`
+const MovieTitle = styled.h4`
+    margin: 4px 0;
+    font-size: 16px;
+    font-weight: 600;
+    color: #fff;
+`
+const MovieDate = styled.p`
+    font-size: 10px;
+    font-weight: 300;
+    color: #fff;
+    margin: 0;
+`
+
+const Card = (props) =>{
```

Comment:
```suggestion
const Card = ({src, title, date, id}) =>{
```

구조 분해 할당으로 미리 쪼개서 받아오시면 props.src이렇게 사용 안하셔도 돼요!